### DISCUSSION TITLE
Open links in new tab

### DISCUSSION THREAD

**Discussion by drvic10k** [Zuehlke/poinz#339] [Ideas]
By default, the links related to the user stories should open in new tab

**Comment by xeronimus** [comment]
Thanks @drvic10k   for the input.

you are right. when Markdown-Mode is turned off, the links in the description of a story are not correctly opened in a new browser tab.
I opened an issue to fix this. (#342 )

Until it is fixed and a new version is released, you can just turn on [markdown mode](https://github.com/Zuehlke/poinz/blob/master/docu/manual.md#adding-stories), then the links will already open in a new tab.
